Source/WebKit/WebProcess/Plugins/PluginView.cpp

```diff
@@ -835,6 +835,7 @@ bool PluginView::performDictionaryLookupAtLocation(const WebCore::FloatPoint& po
 
 void PluginView::notifyWidget(WidgetNotification notification)
 {
+    using enum WidgetNotification;
```

Comment:
I feel like we don't do this too often? Up to you of course.

Reply:
We don't use it too often, yes, though I think a small confined scope like this is ideal for using it since we save some typing, but also there are only two cases for this enum so it doesn't save many characters either.

Reply:
I will drop it from this PR simply because we don't have enough cases to justify "less typing".

Reply:
Fixed in https://github.com/WebKit/WebKit/commit/285179f6316ab2e2595f48f2ea4b965ed48b2ed5